views/svg/uiua.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" viewBox="0 0 32 32" xml:space="preserve" id="svg20" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg"><path id="white" d="m 27.07,7.95 c -1.9,6.7 -5.9,17.8 -11,17.7 -4.5,0 -8.2,-8.1 -11,-17.7 -6.5,8 -0.3,21.8 11,21.7 12.3,-0.2 16.8,-13.8 11,-21.7 z m -11,-5.6 c -5.3,0 -8,2.6 -9.4,3.7 1.1,3.1 4.7,12.6 9.4,12.7 4.7,-0.1 8.3,-9.6 9.4,-12.7 -1.4,-1.1 -4.1,-3.7 -9.4,-3.7 z" fill="#fff" /><path id="outline" fill="#fff" d="M 6.81,6.45 C 9.07,4.2 12.43,2.75 16,2.75 c 3.62,0 6.92,1.48 9.3,3.8 L 25.57,5.78 C 23.17,3.47 19.7,2 16,2 12.3,2 9.01,3.41 6.53,5.69 Z m 20.11,2.03 c 1.48,2.14 2.34,4.72 2.34,7.52 0,7.32 -5.93,13.25 -13.25,13.25 C 8.69,29.25 2.76,23.32 2.76,16 2.76,13.13 3.65,10.51 5.2,8.34 L 4.93,7.43 C 3.03,9.89 2,12.84 2,16 2,23.73 8.27,30 16,30 23.73,30 30,23.73 30,16 30,12.83 28.95,9.91 27.17,7.56 Z" /></svg>
```

Comment:
The `id` should be `uiua`, the fill should be `currentColor` not `white`, then run `make svgo` to optimise the image.